src/index.js

```diff
@@ -1,30 +1,37 @@
 import './style.css';
 import UI from './modules/ui.js';
+import APIScores from './modules/API.js';
 
-const { v4: uuidv4 } = require('uuid');
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+const gameId = 'ewg5IotOpaSFHt0HGmPt';
+const scoresList = document.querySelector('.scores-list');
+const scoreInput = document.querySelector('#score-input');
+const nameInput = document.querySelector('#name-input');
+const refreshButton = document.querySelector('#refresh-button');
+const form = document.querySelector('.form');
 
-const items = [
-  {
-    name: 'Giuseppe',
-    score: '100',
-    id: uuidv4(),
-  },
+// FUNCTIONS
 
-  {
-    name: 'Monacho',
-    score: '12000000000000',
-    id: uuidv4(),
-  },
+async function refreshScores() {
+  const data = await APIScores.getScores(apiUrl, gameId);
+  scoresList.innerHTML = '';
+  data.forEach((item) => UI.renderScoreList(item));
+}
 
-  {
-    name: 'Monocuco10',
-    score: '730',
-    id: uuidv4(),
-  },
-];
+// EVENT LISTENERS
 
-const init = () => {
-  items.forEach((item) => UI.getScoreList(item));
-};
+document.addEventListener('DOMContentLoaded', async () => {
+  refreshScores();
+});
```

Comment:
While you implemented the functionality to communicate with the API correctly, the distribution of that logic in the UI doesn't stick with the project requirements. According to the project requirements, the `Submit` button should `POST` scores to the API, and the `Refresh` button should `GET` the scores from the API and update the UI.

<img width="661" alt="Screen Shot 2022-06-15 at 21 13 33" src="https://user-images.githubusercontent.com/82426682/173976500-d03c1c1a-4c8d-4ec4-b5a2-76256c06d9ae.png">

Please fix this.